unit_tests/test_reactive_handlers.py

```diff
@@ -27,87 +29,157 @@ class TestHandlers(TestCase):
 
     def test_config_chanaged(self):
         """Verify config_changed sets appropriate states."""
-        remove_state('beat.render')
+        remove_state("beat.render")
         config_changed()
-        self.assertTrue(is_state('beat.render'))
+        self.assertTrue(is_state("beat.render"))
 
-    @mock.patch('beats_base.status.waiting')
+    @mock.patch("beats_base.status.waiting")
     def test_waiting_messaging(self, mock_waiting):
         """Verify status is waiting when not connected."""
         waiting_messaging()
         self.assertTrue(mock_waiting.called)
 
-    @mock.patch('beats_base.kv')
+    @mock.patch("beats_base.kv")
+    def test_cache_data(self, mock_kv):
```

Comment:
oh yay!  extra tests  \o/

Reply:
Yes, I have one central cache_data function now so I added new test cases for it